src/Answers/Answers.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+const Answers = (props) => {
+  const correctAnswer = props.question.correct_answer
+  const incorrectAnswers = props.question.incorrect_answers
+  const allAnswers = [...incorrectAnswers, correctAnswer]
```

Comment:
Putting all the answers in one array for displaying them is a great idea! Glad we did it.